bindings/pyroot/src/Pythonize.cxx

```diff
@@ -2691,14 +2691,14 @@ Bool_t PyROOT::Pythonize( PyObject* pyclass, const std::string& name )
 
    }
 
-   else if ( name.substr(0,8) == "TVector3" ) {
+   else if ( name.substr(0,8) == "RVector3" ) {
       Utility::AddToClass( pyclass, "__len__", (PyCFunction) ReturnThree, METH_NOARGS );
       Utility::AddToClass( pyclass, "_getitem__unchecked", "__getitem__" );
       Utility::AddToClass( pyclass, "__getitem__", (PyCFunction) CheckedGetItem, METH_O );
 
    }
 
-   else if ( name.substr(0,8) == "TVectorT" ) {
+   else if ( name.substr(0,8) == "RVectorT" ) {
```

Comment:
Seems like a wrong replacement / change?

Reply:
sure, this is a mistake.